src/sentry/integrations/models/data_forwarding_config.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+from sentry.db.models.fields.jsonfield import JSONField
+
+
+@region_silo_model
+class DataForwardingConfig(DefaultFieldsModel):
+    """
+    Configuration for data forwarding to external services (Segment, SQS, or Splunk).
+    This is a general config that can apply to specific projects or all projects in an organization.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+
+    organization_id = FlexibleForeignKey("sentry.Organization", on_delete=models.CASCADE)
+    is_enabled = models.BooleanField(default=True)
+
+    enroll_new_projects = models.BooleanField(default=False)
+    enrolled_projects = JSONField(default=list)
```

Comment:
models.JSONField is preferred, so lets switch to that one! (e.g. https://github.com/getsentry/sentry/pull/99209)